codecarbon/emissions_tracker.py

```diff
@@ -75,6 +76,13 @@ def __init__(
         elif cpu.is_rapl_available():
             logger.info("CODECARBON : Tracking Intel CPU via RAPL interface")
             self._hardware.append(CPU.from_utils(self._output_dir, "intel_rapl"))
+        
+        # Print warning if no supported hardware is found'
+        if not self._hardware:
+            warnings.warn('CODECARBON : No CPU/GPU tracking mode found. This '\
```

Comment:
A warning sounds good, but let's user the logger instead of the warnings module:
```suggestion
            logger.warning('CODECARBON : No CPU/GPU tracking mode found. This '\
```

Reply:
Perfect @Lothiraldan I did the update on PR as you suggested and removed warnings import as it's unnecessary.

Side-comment, there is a nice explanation in Python docs (https://docs.python.org/3/howto/logging.html) when to use warnings vs logging module for a warning, and it's in line with your suggested change:

"warnings.warn() in library code if the issue is avoidable and the client application should be modified to eliminate the warning

logging.warning() if there is nothing the client application can do about the situation, but the event should still be noted"